cpp/src/groupby/sort/group_rank_scan.cu

```diff
@@ -41,6 +41,37 @@ namespace groupby {
 namespace detail {
 namespace {
 
+template <bool forward, typename permuted_equal_t, typename value_resolver>
+struct unique_identifier {
+  unique_identifier(size_type const* labels_,
```

Comment:
As described in the [developer guide](https://docs.rapids.ai/api/libcudf/stable/developer_guide), the internal variables should be prefixed with `_` and the parameters coming into the function should not.

Reply:
Sorry, that's my bad I forgot about that section of the guide. I will update this everywhere, thanks for pointing it out